assets-ph1-website/index.html

```diff
@@ -85,21 +86,21 @@
 
   <footer>
     <div class="footerInner">
-      <div class="line1"></div>
-      <div class="logo_footer"><img src="./img/logo.svg" alt="logo"></div>
-      <nav class="link_footer">
+      <div class="footer_line1"></div>
```

Comment:
```suggestion
      <div class="footer_line"></div>
```

必要あるかな？
javascriptで利用する目的だとしたら
class名に`js-`をつけてあげると意味があるのがわかりやすいです